### PR TITLE
index: order by id

### PR DESCRIPTION
**Pull request by grvcoelho**
## Description

Order items by id on index.

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](../CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation
1. I feel confortable with the code I wrote, and I'm not ashamed to show it to my friends

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

File: src/resources/boleto/service.ts [hunk 1]
```diff
@@ -119,6 +119,10 @@ export const index = ({ page, count, token, title_id }) => {
     }
   }
 
+  const orderQuery = {
+    order: 'id DESC'
```
Comment (marquinhoalm) [inline src/resources/boleto/service.ts]: Tem que ser um `order: [['id, 'desc']]`

Comment (grvcoelho) [inline src/resources/boleto/service.ts]: Por que?

Comment (marquinhoalm) [inline src/resources/boleto/service.ts]: funciona do jeito que colocasse também.. só passei a estrutura correta do que tinha colocando antes e estava dando erro =)

File: src/resources/boleto/service.ts [hunk 2]
```diff
@@ -129,7 +133,13 @@ export const index = ({ page, count, token, title_id }) => {
   }
 
   const paginationQuery = getPaginationQuery({ page, count })
-  const query = mergeAll([{}, paginationQuery, whereQuery])
+
+  const query = mergeAll([
+    {},
```
Comment (mccraveiro) [inline src/resources/boleto/service.ts]: Why do we have this empty object?

**Pull request merged**